lib/routes/imop/tianshu.ts

```diff
@@ -0,0 +1,54 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+import iconv from 'iconv-lite';
+import cache from '@/utils/cache';
+
+const baseUrl = 'http://t.imop.com';
+
+export const route: Route = {
+    path: '/tianshu/:page?',
+    categories: ['game'],
+    example: '/imop/tianshu/all',
+    radar: [
+        {
+            source: ['t.imop.com'],
+            target: '/tianshu',
+        },
+    ],
+    name: '全部消息',
+    maintainers: ['zhkgo'],
+    handler,
+};
+
+async function handler(ctx) {
+    const page = ctx.req.param('page') || 1;
```

Comment:
No page turn. Returning the content from the first page is enough.

Reply:
yes, the first page is not zero. I changed the example of route.